<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferSavesDialog</class>
 <widget class="QDialog" name="TransferSavesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transfer Save Games</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Global Characters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="globalCharacterList">
       <property name="toolTip">
        <string>This is a list of characters in the global location.</string>
       </property>
       <property name="whatsThis">
        <string>This is a list of characters in the global location.

On Windows Vista/Windows 7:
  C:\Users\[UserName]\Documents\My Games\Skyrim\Saves

On Windows XP:
  C:\Documents and Settings\[UserName]\My Documents\My Games\Skyrim\Saves
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="globalSavesList">
       <property name="whatsThis">
        <string>This is a list of save games for the selected character in the global location.

On Windows Vista/Windows 7:
  C:\Users\[UserName]\Documents\My Games\Skyrim\Saves

On Windows XP:
  C:\Documents and Settings[UserName]\My Documents\My Games\Skyrim\Saves

</string>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="moveToLocalBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Move -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyToLocalBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Copy -&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="moveToGlobalBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;- Move</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyToGlobalBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&lt;- Copy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>100</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="doneButton">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Profile Characters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="localCharacterList"/>
     </item>
     <item>
      <widget class="QListWidget" name="localSavesList">
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
